level1.cc

```diff
@@ -74,22 +74,22 @@ int Level1::pickMove() {
     std::random_device dev;
     std::mt19937 rng(dev());
     std::uniform_int_distribution<std::mt19937::result_type> randompick(0, size-1); //picks random index in newMap
-    std::map<Piece *, Box> iterator item = newMap.begin();                //iterator points to {Piece*:Box}
+    std::map<Piece *, Box>::iterator item = newMap.begin();                //iterator points to {Piece*:Box}
```

Comment:
Is adding the `::` a change that needs to be made in creating each iterator that goes through a map?

Reply:
yeah `std::map<Piece *, Box> iterator item` is improper notation.